config/routes.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  root 'users#index'
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show]
```

Comment:
- [ ] [OPTIONAL] You have done well by implementing some arrangements but it's equally important that you mind the alignment. Can you keep your User interfaces all centralized just as we have on the wireframes contained in the screenshots above? :thinking: 
- Evening the whitespace will definitely improve user experience and make the app look more interesting to use.

![image](https://github.com/shm04/blogApp/assets/99715640/42df253e-250f-4ac3-a96e-bd74ebc6a24d)